### PR TITLE
add perfectgonzo.yml

### PR DESCRIPTION
**Pull request by JaseNZC**

### CODE REVIEW

**Comment by JaseNZC** [comment]
Fixed my stupid performer error.

File: scrapers/perfectgonzo.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - perfectgonzo.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right"]/h2
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:11]: a little safer
```yaml
        selector: //div[@class="row"]//h2
```

File: scrapers/perfectgonzo.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      Date:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right text-right"]/span
+        postProcess:
+          - replace:
+              - regex: 'Added|,'
+                with: ''
+          - parseDate: January 02 2006
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:18]: For single digit days there is no 0 in  the site so it should be `2` and not `02`
Also `,` can be parsed so its not necessary to remove it
Safer to use
```yaml
      Date:
        selector: //div[@class="row"]//span
        postProcess:
          - replace:
              - regex: "Added "
                with: ""
          - parseDate: January 2, 2006
```

File: scrapers/perfectgonzo.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      Details: //div[@class="col-sm-8 col-md-8 no-padding-side"]/p
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:19]:
```yaml
      Details: //p[@class="mg-md"]
```

File: scrapers/perfectgonzo.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      Tags:
+        Name: //div[@class="col-sm-8 col-md-8 no-padding-side tag-container"]//a
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:21]:
```yaml
        Name: //div[contains(h4,"Tags:")]//a
```

File: scrapers/perfectgonzo.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      Studio:
+        Name:
+          fixed: Perfect Gonzo
+      Image: //video[@id="video"]/@poster | //div[@class="container bloc-sm"]//@src
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:25]: Do you have the URL of the scene that the second part is needed?
The second part should be more restrictive as it seems to return a lot of links inlcluding videos

File: scrapers/perfectgonzo.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      Performers:
+       Name: //div[@class="col-sm-3 col-md-3 col-md-offset-1 no-padding-side"]//a
```
Comment (bnkai) [inline scrapers/perfectgonzo.yml:27]:
```yaml
        Name: //div[contains(h4,"Featured model")]//a
```

**Comment by JaseNZC** [comment]
Thanks for all the help I am learning as I go.

File: scrapers/perfectgonzo.yml (hunk 5 above)
Comment (JaseNZC) [reply scrapers/perfectgonzo.yml:25]: Any scene that is an interview as a different image layout.
[https://www.perfectgonzo.com/movies/dolly-diore-interview/?nats=](url)
Having a look it should have been 
class="container col-sm-12"

Comment (bnkai) [reply scrapers/perfectgonzo.yml:25]:
```yaml
      Image: //video[@id="video"]/@poster | //div[@id="video-hero"]//img/@src
```
seems to get the job done and is probably more future proof
can you verify against some of your samples ? (works for me for a few scenes)

Comment (JaseNZC) [reply scrapers/perfectgonzo.yml:25]: Yup works on all scenes , you can make the change :-)

**Pull request merged**